config/config.go

```diff
@@ -44,9 +44,12 @@ type Config struct {
 	ServerAddress string `default:":5050" split_words:"true"`
 	Secret        []byte `split_words:"true"`
 
-	TurnAddress    string `default:":3478" required:"true" split_words:"true"`
-	TurnStrictAuth bool   `default:"true" split_words:"true"`
-	TurnPortRange  string `split_words:"true"`
+	TurnAddress            string `default:":3478" required:"true" split_words:"true"`
+	TurnStrictAuth         bool   `default:"true" split_words:"true"`
+	TurnPortRange          string `split_words:"true"`
+	TurnExternal           bool   `default:"false" split_words:"true"`
+	TurnCoturnSignedTtl    string `default:"12h" split_words:"true"`
+	TurnCoturnSignedSecret string `split_words:"true"`
```

Comment:
I would suggest avoiding using "coturn" in config name, how about `TurnExternalSecret`?

To be honest I am not sure if we even need to expose TTL param, we could try to be pragmatic and leave it hardcoded to 24h as suggested in the [docs](http://tools.ietf.org/html/draft-uberti-behave-turn-rest-00) and wait until someone actually has a need to configure this? 🤔

Reply:
coturn seems to be the only TURN server using the hmac trick for generating password.

* [ ] Hardcode TTL to 24h

Reply:
Since coturn is the leading implementation I think they are the only ones who do many things 😄 I just thought since it's a proposed standard for REST authentication, documented not on coturn's website but on ietf.org, others _might_ implement this as well at some point 🤷‍♂️

Reply:
Fixed in 7ba0af398f0c60351c1df022c38689900a05e59b

Reply:
Awesome, thanks! Super excited to see this PR becoming more and more finished 👍 

Just for my understanding, I'm very new to TURN, I see you use "signed secret" term, does that mean that the shared secret is already signed by coturn, or that we sign something else using this secret? It's not a comment, I'm just curious how this works internally, coturn calls this value "auth secret" and the doc you mentioned in the first message calls it "shared secret key", so I would assume that this secret is not signed, but is used for signing?